### PR TITLE
Add info about Che Github integration

### PR DESCRIPTION
**Pull request by david-martin**

### CODE REVIEW

**Comment by david-martin** [comment]
@odra would you mind reviewing?

File: che/auth.md [hunk 1]
```diff
@@ -9,4 +9,18 @@
...
+# Github Integration
+
+Che has the ability to integrate with Github, providing 2 main features:
+
+* Import a git repo as a Che Project (shows a list of user's repos)
+* Do version control from the Che IDE
+
+There are 2 ways to configure Che server for Github integration.
+
+* Set env vars in Che server for a github client id & secret (embedded/single user mode)
```
Comment (odra) [inline che/auth.md]: can you add the env vars names here: CHE_OAUTH_GITHUB_CLIENTID and CHE_OAUTH_GITHUB_CLIENTSECRET?

**Comment by odra** [comment]
@david-martin looks good, just a comment about the env vars section.

**Comment by david-martin** [comment]
@odra added env vars

**Review by odra — APPROVED** [review]

**Pull request merged**